### ISSUE TITLE
Error when installing .cia file through FBI

### ISSUE THREAD

**Issue by lilbud** [PabloMK7/ArticBaseServer#3]
Every time I attempt to install ArticBase it fails, I've downloaded it a few times and each time it fails with the same error on FBI. I also tried remote install, but same error codes.

Result Code: 0xD9004588
Level: Permanent (27)
Summary: Wrong argument (8)
Module: FS (17)
Desc: RSA/Hash verification failure (392)

Hardware: Old 3DS XL (NA Region, not sure if that matters)
Firmware: 11.17.0-50U

Software versions:
Luma 13.1.1
FBI 2.6.1

I had to install a few .cia files with FBI before hand, so I know that my SD card works and so does FBI. I also updated Luma to the latest version as well.

Unsure if I'm supposed to use something else to install Articbase instead?

**Comment by PabloMK7** [comment]
Your SD card is corrupted. Backup everything to your computer, format the SD and copy everything back. Then download the file again and try to install it.

**Comment by lilbud** [comment]
Okay, I formated the SD card. I also ran the "h2testw" tool linked [here](https://wiki.hacks.guide/wiki/Checking_SD_card_integrity/Windows) to test the integrity of my SD card, it passed and showed no signs of error.

I copied everything back, and made sure everything was up to date as well. 

I then redownloaded the .cia file from the releases page, copied to my SD card, and attempted another install with FBI. I still get the same error listed above.

I managed to find another SD card, went through the above process again (format, test, recopy files). Tried another install and still get the same error. I don't have issues installing other .cia files, just seems to be ArticBase for unknown reasons.
